### PR TITLE
Add filename for dynamic rarity 

### PR DESCRIPTION
**Pull request by cecpk**
Adding a argument for the dynamic rarity file. If empty = rarity.json (no changes to the current state)

## Description
Add argument -RFN for dynamic Filename - if empty filename=rarity

## Motivation and Context
Using two different RM Databases with two instances out of the same folter caused problems in showing Pokemons with dynamic rarity. One instance overwrite the rarity from the other. 

## How Has This Been Tested?
On my own forks.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pogom/app.py [hunk 1]
```diff
@@ -359,7 +359,8 @@ def fullmap(self, statusname=None):
                                generateImages=str(args.generate_images).lower(),
                                gmaps_key=args.gmaps_key,
                                lang=args.locale,
-                               show=visibility_flags
+                               show=visibility_flags,
+				rarityFileName=args.rarityfilename
```
Comment (rlodev67) [inline pogom/app.py]: Didn't tested it, but it won't respect the indentation... I think it will not compile

File: pogom/utils.py [hunk 2]
```diff
@@ -579,6 +579,10 @@ def get_args():
                               ' should be updated. Decimals allowed.' +
                               ' Default: 0. 0 to disable.'),
                         type=float, default=0)
+    parser.add_argument('-RFN', '--rarityfilename', type=str,
```
Comment (rlodev67) [inline pogom/utils.py]: Please use --rarity-filename and -Rfn

**Comment by rlodev67** [comment]
And btw please update to documentation too and config.ini.example

**Comment by CRAKD** [comment]
I didn’t even realize this was happening. I run 4 os instances out of the same folder so it’s definitely happening for me.

**Comment by cecpk** [comment]
it´s my first strike :)
i changes this - thx

File: pogom/app.py [hunk 3]
```diff
@@ -359,7 +359,8 @@ def fullmap(self, statusname=None):
                                generateImages=str(args.generate_images).lower(),
                                gmaps_key=args.gmaps_key,
                                lang=args.locale,
-                               show=visibility_flags
+                               show=visibility_flags,
+			       rarityFileName=args.rarity_filename
```
Comment (rlodev67) [inline pogom/app.py]: Same, please indent it correclty (don't use tab, use spaces)

File: pogom/utils.py [hunk 4]
```diff
@@ -579,6 +579,10 @@ def get_args():
                               ' should be updated. Decimals allowed.' +
                               ' Default: 0. 0 to disable.'),
                         type=float, default=0)
+    parser.add_argument('-Rfn', '--rarity-filename', type=str,
+                        help=('Filename of Rarity Json for different Databases (without .json)' +
```
Comment (rlodev67) [inline pogom/utils.py]: Filename of rarity json for different databases (without .json)

File: config/config.ini.example [hunk 5]
```diff
@@ -142,6 +142,8 @@
 #disable-blacklist              # Disable the global anti-scraper IP blacklist.
 #rarity-hours:                  # Number of hours of Pokémon data to use to calculate dynamic rarity, decimals allowed. 0 to use all data. (default=48.0)
 #rarity-update-frequency:       # How often (in minutes) the dynamic rarity should be updated. decimals allowed. 0 to disable. (default=0)
+#rarity-filename:               # Name of the JSON File for dynamic rarity. Using two different RM Databases with two instances out of the same folter. Default (if empty): ra.rarity.json
+
```
Comment (rlodev67) [inline config/config.ini.example]: According to utils.py code, default is "rarity"
Maybe you can add without ".json" in the example

**Pull request closed without merging**